.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,43 @@
+name: unit-tests
+on:
+  push:
+    branches:
+      - "master"
+      - "test-me/*"
+  pull_request:
+    branches:
+      - "*"
+  schedule:
+    - cron: "0 7 * * 1" # Run every Monday at 7:00 UTC
```

Comment:
Don't you need to specify here which branches need to run the cron-job? Only master, I assume?

Reply:
Crons only run on `master`